tests/test_model.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+from hydrolib.core.basemodel import FileModel
+from hydrolib.core.models import DIMR, FMModel
+
+from .utils import test_data_dir, test_output_dir
+
+
+def test_filemodel_load():
+    d = DIMR(filepath="a", model=Path(test_data_dir / "test.mdu"))
```

Comment:
Did you intend to test with a file that does not exist?

Reply:
Yes, I will make this explicit in the test description.